src/components/member-card/index.js

```diff
@@ -92,10 +118,20 @@ Card.propTypes = {
     img_url: PropTypes.string.isRequired,
     isMember: PropTypes.bool.isRequired,
   }).isRequired,
+
   isOptionKey: PropTypes.bool,
+
+  colorCombination: PropTypes.shape({
+    color_primary: PropTypes.string.isRequired,
+    color_secondary: PropTypes.string.isRequired,
+  }),
 };
 Card.defaultProps = {
   isOptionKey: false,
+  colorCombination: {
+    color_primary: '#DB1212',
+    color_secondary: '#F88181',
```

Comment:
can we move these constants to a constat file and then import them here and use it? In the future, we might need these default colors elsewhere also. What do you think?